raiden/transfer/token_network.py

```diff
@@ -30,64 +30,90 @@
 ]
 
 
-def subdispatch_to_channel_by_id_and_address(
+def subdispatch_to_channels_by_participant_address(
```

Comment:
I like this because: 

- we were not using channel id
- participant seems better than address

but.. im a little bit concern about the usage, we should test basically all the onchain operations manually and run the test suite

Reply:
Agree, I'll run the tests at the end of the day. 
The method is only used internally in this file and seems like it hadn't break anythink that wasn't already broken (channel close and update balance proof) but would be wrong not to run the test suite